src/routes/safe/components/Balances/Coins/index.tsx

```diff
@@ -30,10 +27,29 @@ import {
   getBalanceData,
 } from 'src/routes/safe/components/Balances/dataFetcher'
 import { extendedSafeTokensSelector, grantedSelector } from 'src/routes/safe/container/selector'
+import { Skeleton } from '@material-ui/lab'
 
 const useStyles = makeStyles(styles as any)
 
-const Coins = (props) => {
+type Props = {
+  showReceiveFunds: () => void
+  showSendFunds: (tokenAddress: string) => void
+}
+
+export type BalanceDataRow = List<{
```

Comment:
this is defined in `src/routes/safe/components/Balances/dataFetcher.ts` and it's expected for the type to be there too